django/db/models/fields/related.py

```diff
@@ -1314,7 +1314,7 @@ class ForeignObjectRel(object):
 
     # Reverse relations are always nullable (Django can't enforce that a
     # foreign key on the related model points to this model).
-    null = True
+    null = False
```

Comment:
It's not defined as a class attribute, but [it was being set at runtime](https://github.com/django/django/pull/5202/files#diff-3010fc5a498b7171c342520f34507968L2308)

Also note that this change ought to include the comment above this line (or remove this line in favor of leaving the `__init__` setting, as before.)